View All not working fix

mrabhishekchauhan opened the pull request:
Bug No- 108
Issue - view all was not working on the app
RCA - searchQuery key was not there in the response for getHomePage to the app
Fix - added the searchQuery key and its value for view all to work
addresses issue : https://project-sunbird.atlassian.net/projects/OP/issues/OP-108

mrabhishekchauhan committed:
add TextBook key to fix missing headline key

mrabhishekchauhan committed:
Merge branch 'develop' of https://github.com/projectOpenRAP/Diksha into diksha/fix-missing-headline/op-21

mrabhishekchauhan committed:
plugin fix  to support changing config and homePage file

mrabhishekchauhan committed:
Merge branch 'develop' of https://github.com/projectOpenRAP/Diksha into diksha/diksha-plugin-fix/OP-80

mrabhishekchauhan committed:
indentation fix

mrabhishekchauhan committed:
minor changes

mrabhishekchauhan committed:
View all not working fix

mrabhishekchauhan committed:
Merge branch 'develop' of https://github.com/projectOpenRAP/Diksha into diksha/view-all-fix/OP-108

rak16 commented:
1. Aren't these changes already a part of https://github.com/projectOpenRAP/Diksha/pull/6 ?
2. You are supposed to create a PR in `develop`, not master.

mrabhishekchauhan commented:
1) i removed those changes from that commit, this is a different bug and to keep track of it separately.
2) changed the base from master to develop

Skeletrox commented:
Just add relevant fixes
